src/components/SidebarMobile/SidebarMobile.jsx

```diff
@@ -40,15 +39,15 @@ export default class SidebarMobile extends React.Component {
 
   componentDidMount() {
     if (typeof window !== 'undefined') {
-      // window.addEventListener('click', this._handleBodyClick);
-      window.addEventListener('touchstart', this._handleBodyClick);
+      window.addEventListener('mousedown', this._handleBodyClick);
+      // window.addEventListener('touchstart', this._handleBodyClick);
```

Comment:
Not necessary?

Reply:
Doesn't seem to be. But I've only been able to test with emulated mobile in chrome dev tools

Reply:
Uhmm, we need to test in real devices